jetty-core/jetty-server/src/main/java/module-info.java

```diff
@@ -28,4 +28,5 @@
 
     exports org.eclipse.jetty.server.jmx to
          org.eclipse.jetty.jmx;
+    exports org.eclipse.jetty.server.internal;
```

Comment:
Nope... dang intellij!

Reply:
I do find the whole jpms things strange.... previously I would have just made this class package private and it would have worked fine... but now I have to move it into a different package 'internal' and make it public so I can use it, but then use a module.info file to restrict who can use it.    Not sure I see the value in a public/restricted org.foo.internal.Bar vs package protected org.foo.Bar?